src/FxKit.CompilerServices/FxKit.CompilerServices.csproj

```diff
@@ -6,6 +6,7 @@
 
     <Title>FxKit Compiler Services</Title>
     <Description>Provides Roslyn source generators and analyzers for FxKit data types.</Description>
+    <DevelopmentDependency>true</DevelopmentDependency>
```

Comment:
I haven't seen this flag before, what does it do? I thought `PrivateAssets` was sufficient.